src/components/Explore/TokenTable/TokenTable.tsx

```diff
@@ -46,11 +46,6 @@ const NoTokenDisplay = styled.div`
 const TokenRowsContainer = styled.div`
```

Comment:
this whole file is just little code quality fixes that I made while thinking about loading states.